LanguageTranslatorV3.playground/Pages/Document translation.xcplaygroundpage/Contents.swift

```diff
@@ -54,8 +54,8 @@ languageTranslator.getDocumentStatus(documentID: testDocumentID) {
 languageTranslator.deleteDocument(documentID: testDocumentID) {
     response, error in
     
-    if (response?.statusCode != 204) {
-        print(error?.localizedDescription ?? "unknown error")
+    if let error = error {
+        print(error.localizedDescription ?? "unknown error")
```

Comment:
Actually you don't need the `?? "unknown error"` here either, since `error.localizedDescription` is not an optional.